packages/devtools_app/test/vm_developer/vm_developer_test_utils.dart

```diff
@@ -0,0 +1,244 @@
+import 'package:devtools_app/src/screens/debugger/debugger_model.dart';
+import 'package:devtools_app/src/screens/vm_developer/object_inspector_view_controller.dart';
+import 'package:devtools_app/src/screens/vm_developer/object_viewport.dart';
+import 'package:devtools_app/src/screens/vm_developer/vm_object_model.dart';
+import 'package:flutter/foundation.dart';
+import 'package:vm_service/vm_service.dart';
+
+final fakeLibRef =
+    TestLibraryRef(name: 'fooLib', uri: 'fooLib.dart', id: '1234');
+
+final fakeLib = Library(
```

Comment:
for any var that isn't technically a "fake" (i.e. any that are an instance of a real class from the vm service), do not include "fake" in the name. We can call these 'testLib', 'testClass' etc.